src/docfx/validation/JsonSchemaValidatorExtension.cs

```diff
@@ -37,5 +39,31 @@ public bool IsEnable(FilePath filePath, CustomRule customRule)
 
             return customRule.ContentTypes is null || customRule.ContentTypes.Contains(pageType);
         }
+
+        public List<string> GetFileEffectiveMonikers(FilePath filePath, CustomRule? customRule)
```

Comment:
consider add an optional `moniker` property to `IsEnabled` method because there are other filtering logic in the `IsEnabled` method and we'll likely add more.